playwright/_impl/_network.py

```diff
@@ -263,16 +285,34 @@ def status_text(self) -> str:
 
     @property
     def headers(self) -> Dict[str, str]:
-        return parse_headers(self._initializer["headers"])
+        return self._headers.copy()
+
+    async def all_headers(self) -> Dict[str, str]:
+        return headers_array_to_object(await self._get_headers_if_needed(), True)
+
+    async def headers_array(self) -> List[List[str]]:
+        return list(
+            map(
+                lambda header: [header["name"], header["value"]],
```

Comment:
Better to make it a tuple

Reply:
we want to keep it the same in all the languages.